### DISCUSSION TITLE
Can we assign a value to empty variables?

### DISCUSSION THREAD

**Discussion by Nebucatnetzer** [docker-mailserver/docker-mailserver#2888] [Q&A] [answered]
Some of the values expect empty to disable them.
Can we use something like `variable=null` or `variable=""` to achieve the same?

**Comment by polarathene** [comment, marked as answer]
An empty string should be a valid approach if the docs for that environment variable use no value / empty as a way to disable / opt-out.

**Comment by Nebucatnetzer** [comment]
Good to know, thank you!
